server/api.js

```diff
@@ -88,9 +88,12 @@ async function messageClients(req, res) {
       const message = req.body.message
       const response = {
         status: 'success',
-        message,
-        contacted: [],
-        failed: [],
+        data: {
+          message,
+          contacted: [],
+          failed: [],
+        },
+        message: '',
       }
```

Comment:
make sure the format of the data provided here matches the structure expected by the lambda function handling your POST request. you might need to adjust the lambda function to handle the data in the same format as provide here. you have added new fields and there should be accounted for in the Lambda function as well.

Reply:
Hi Ahmad, which Lambda function are you referring to? What I have changed is the information that is sent in response to a successful POST request. The information POSTed to this API call has not changed.

At present, the response information is not used anywhere; it will be handled in PA, which I will work on in a future PR.